src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -299,6 +312,43 @@
         <DataTemplate x:Key="EmptyTemplate" x:DataType="ViewModels:PluginAdditionalOptionViewModel">
             <StackPanel HorizontalAlignment="Stretch" Orientation="Vertical" />
         </DataTemplate>
+
+        <DataTemplate x:Key="MetadataVersionTemplate" x:DataType="ViewModels:PluginMetadataViewModel">
+            <TextBlock Style="{ThemeResource SecondaryTextStyle}">
+                <Run x:Uid="PowerLauncher_PluginVersion" />
+                <Run FontWeight="SemiBold" Text="{x:Bind Value}" />
+            </TextBlock>
+        </DataTemplate>
+
+        <DataTemplate x:Key="MetadataAuthorTemplate" x:DataType="ViewModels:PluginMetadataViewModel">
+            <TextBlock Style="{ThemeResource SecondaryTextStyle}">
+                <Run x:Uid="PowerLauncher_AuthoredBy" />
+                <Run FontWeight="SemiBold" Text="{x:Bind Value}" />
+            </TextBlock>
+        </DataTemplate>
+
+        <DataTemplate x:Key="MetadataWebsiteTemplate" x:DataType="ViewModels:PluginMetadataViewModel">
+            <HyperlinkButton
+                Margin="0"
+                Padding="0"
+                AutomationProperties.Name="{x:Bind Value}"
+                NavigateUri="{x:Bind Value}"
+                Style="{StaticResource TextButtonStyle}"
+                ToolTipService.ToolTip="{x:Bind Value}">
```

Comment:
Don't think it's common to show the url in the tooltip across the OS? I'd suggest to remove it to not getting any a11y flags because the narrator / tooltip values are different.

Reply:
But then users blind open urls we can't control? Security problem?